samples/peripheral/802154_phy_test/prj.conf

```diff
@@ -43,3 +43,7 @@ CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE=y
 
 # Enable ring buffers
 CONFIG_RING_BUFFER=y
+
+CONFIG_NRF_802154_TEMPERATURE_UPDATE=n
+
+CONFIG_NRF_802154_ENCRYPTION=n
```

Comment:
I understand that 802.15.4 encryption is disabled, because it is a MAC layer feature that is not needed for PHY testing (it could make some tests hard). But why temperature update is disabled? I think RSSI temperature correction is essential for some of PHY layer features like setting correct Energy Detection threshold, or reporting correct link margin value in broad temperature range.

Reply:
This is a hotfix for a problem, that as I have been informed by @pkwiek is a known issue. Without this, the PHY Test Tool won't work, as it enters and infinite ASSERT FAILURE and restart loop.

Reply:
Let me shine a bit of light on this: initial implementation of PTT did not use the SENSOR subsystem to obtain the data, but rather probed for the temperature explicitly using the `mpsl_...` call. This simplified approach was chosen to faciliate the use of such information on nRF5340-NET platform. The correct re-implementation using SENSOR was planned altogether with porting to nRF5340-APP platform (where some kind of serialization would be needed), but as we know, exporting this sample to the NCS happened earlier.

Reply:
So what's the plan now? To use `mpsl_...` calls in PTT? Is it implemented? Does PTT add correction to RSSI measurements? Or just provide temperature getter command?